heft-plugins/heft-vscode-extension-plugin/src/VSCodeExtensionPublishPlugin.ts

```diff
@@ -0,0 +1,94 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+import type {
+  HeftConfiguration,
+  IHeftTaskPlugin,
+  IHeftTaskSession,
+  IHeftTaskRunHookOptions,
+  CommandLineStringParameter
+} from '@rushstack/heft';
+import type { IWaitForExitResult } from '@rushstack/node-core-library';
+import * as path from 'node:path';
+import { execuateAndWaitAsync, vsceScriptPath } from './util';
```

Comment:
```suggestion
import { executeAndWaitAsync, vsceScriptPath } from './util';
```
Fix the function name